Source/WebKit/Platform/cocoa/XPCUtilities.mm

```diff
@@ -26,17 +26,52 @@
 #include "config.h"
 #include "XPCUtilities.h"
 
+#include <wtf/WTFProcess.h>
+
 namespace WebKit {
 
+static auto constexpr messageNameKey = "message-name";
+static auto constexpr exitProcessMessage = "exit";
```

Comment:
Please don't use `const char*` for string literals in new code. This should use `"exit"_s`

Reply:
Ah, good catch, fixed!

Thanks for reviewing!